scripts/ClientSetup.ps1

```diff
@@ -158,3 +187,128 @@ if ($ServiceSalt) {
 }
 choco feature enable --name="'useChocolateyCentralManagement'"
 choco feature enable --name="'useChocolateyCentralManagementDeployments'"
+
+
+Write-Host "Applying user supplied configuration" -ForegroundColor Cyan
+#How we call choco from here changes as we need to be more dynamic with thingsii .
+if ($AdditionalConfiguration) {
+    <#
+    We expect to pass in a hashtable with configuration information with the following shape:
+
+    @{
+        Name = BackgroundServiceAllowedCommands 
+        Value = 'install,upgrade,uninstall'
+    }
+    #>
+
+    $AdditionalConfiguration.GetEnumerator() | ForEach-Object {
+        $c = [System.Collections.Generic.list[string]]::new()
+        $c.Add('config')
+        $c.Add('set')
+        $c.Add("--name='$($_.Key)'")
+        $c.Add("--value='$($_.Value)'")
+
+        & choco @c
+    }
+}
+
+if ($AdditionalFeatures) {
+    <#
+    We expect to pass in feature information as a hashtable with the following shape:
+
+    @{
+        Name = useBackgroundservice
+        State = 'Enabled'
+    }
```

Comment:
It looks like this comment-help is outdated, as I think the code implies it should be something like this:
```suggestion
    @{
        "useBackgroundservice" = 'Enabled'
    }
```